src/main/webapp/courseLoader.js

```diff
@@ -0,0 +1,124 @@
+google.charts.load("current", { packages: ["corechart"] });
+google.charts.setOnLoadCallback(makeGraphs);
+
+function fillTitles() {
+    const queryString = window.location.search;
+    const urlParams = new URLSearchParams(queryString);
+    const courseName = urlParams.get("course-name");
+    const term = urlParams.get("term");
+    document.getElementById("course-name").innerHTML = courseName;
+    document.getElementById("term-name").innerHTML = term;
+}
+
+function populateData(){
+    const queryString = window.location.search;
+    const urlParams = new URLSearchParams(queryString);
+    const courseName = urlParams.get("course-name");
+    const term = urlParams.get("term");
+    const profName = urlParams.get("prof-name");
+    const units = urlParams.get("num-units")
+    const schoolName = urlParams.get("school-name")
+
+  fetch(`/term-live?school-name=${schoolName}&course-name=${courseName}&term=${term}&prof-name=${profName}&num-units=${units}`)
```

Comment:
Are *all* of these query params required to specify the desired term?

Reply:
Sadly with the current implementation yes. I think soon we can modify how the url is sent to me so it has the entities actual key which would make it simpler.